logout.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials
+
+import (
+	"context"
+	"fmt"
+)
+
+func Logout(ctx context.Context, store Store, registryName string) error {
+	if err := store.Delete(ctx, registryName); err != nil {
```

Comment:
Should we redirecting `docker.io` like oras does? See https://github.com/oras-project/oras/blob/main/cmd/oras/root/logout.go#L64-L70. /cc @qweeah